lib/app_profiler/middleware.rb

```diff
@@ -4,7 +4,6 @@
 require "app_profiler/middleware/base_action"
 require "app_profiler/middleware/upload_action"
 require "app_profiler/middleware/view_action"
-require "app_profiler/sampler/config"
```

Comment:
I don't think we meant to remove this?

cc @bmansoob

Reply:
I did, it is autoloaded now because the profile called the deprecated but was required after the deprecator was defined. So I allowed all toplevel constants to autoload (like this one, `AppProfiler::Sampler`).

Reply:
Ah makes sense, didn't see that it moved to the autoload 👍  just wanted to make sure we didn't inadvertently add a regression.